GSA-TTS/FAC
# Adding workbook version to sac records

## Description
We do not currently record the schema version with SAC records, and while this has been intended for a while, it was not prioritized. Recently, we encountered production issues where users failed validation due to using different workbook template versions. After resolving these issues, the team decided that recording the workbook validation schema version with the SAC record is now a must-have.

This ticket is to implement the logic for capturing the schema version along with the SAC record.

## How to Test
1. Launch the application locally from the main branch.
2. Start a submission and upload some or all workbooks (preferably from different versions).
3. Check the SAC record in the database, and note that the section columns does not include a version.
4. Switch to this PR branch and re-upload the same files.
5. Check the database and confirm that the sections now include the version number matching the workbook version.
## PR Checklist: Submitter


## PR Checklist: Reviewer


The larger the PR, the stricter we should be about these points.

## Pre Merge Checklist: Merger
